tgui/packages/tgui/interfaces/Fax.tsx

```diff
@@ -81,48 +81,52 @@ export const Fax = (props) => {
           </LabeledList.Item>
         </Section>
         <Section title="Send">
-          {faxes.length !== 0 ? (
+          {faxes.length === 0 && data.special_faxes.length === 0 ? (
+            "The fax couldn't detect any other faxes on the network."
+          ) : (
             <Box mt={0.4}>
-              {(data.syndicate_network
-                ? data.special_faxes
-                : data.special_faxes.filter(
-                    (fax: FaxSpecial) => !fax.emag_needed,
-                  )
-              ).map((special: FaxSpecial) => (
-                <Button
-                  key={special.fax_id}
-                  tooltip={special.fax_name}
-                  disabled={!data.has_paper}
-                  color={special.color}
-                  onClick={() =>
-                    act('send_special', {
-                      id: special.fax_id,
-                      name: special.fax_name,
-                    })
-                  }
-                >
-                  {special.fax_name}
-                </Button>
-              ))}
-              {faxes.map((fax: FaxInfo) => (
-                <Button
-                  key={fax.fax_id}
-                  tooltip={fax.fax_name}
-                  disabled={!data.has_paper}
-                  color={fax.syndicate_network ? 'red' : 'blue'}
-                  onClick={() =>
-                    act('send', {
-                      id: fax.fax_id,
-                      name: fax.fax_name,
-                    })
-                  }
-                >
-                  {fax.fax_name}
-                </Button>
-              ))}
+              {data.special_faxes.length !== 0
+                ? (data.syndicate_network
+                    ? data.special_faxes
+                    : data.special_faxes.filter(
+                        (fax: FaxSpecial) => !fax.emag_needed,
+                      )
```

Comment:
might be out of scope so this is optional,
but can this please be made a constant outside of the jsx block like is done with `faxes`